src/publishing/s3publisher.py

```diff
@@ -113,14 +113,6 @@ def publish_to_s3(directory, base_url, site_prefix, bucket, federalist_config,
 
                 local_objects_by_filename[site_file.filename] = site_file
 
-                if filename == 'index.html':
-                    site_redirect = SiteRedirect(filename=root,
```

Comment:
Removing `SiteRedirect` may break some users' site links. Without the redirect, a link to `/my-page` may break because it isn't actually going to the expected`/my-page/`. What was your thoughts behind removing `SiteRedirect`?

Reply:
I'll try to reconstruct my thoughts at the time but I think it was harder to cache these objects and I thought they weren't necessary because https://github.com/cloud-gov/pages-proxy/issues/113 was closed but it looks like it wasn't implemented. I'll revisit now

Reply:
Sounds good. Maybe we can look through some build logs to see how often this is used.

Reply:
I messed a bit too much with the git history but this should be fixed now (cache-control added to `SiteRedirect` also)